runtime/development/src/lib.rs

```diff
@@ -1032,6 +1032,7 @@ parameter_types! {
 impl pallet_pool_system::Config for Runtime {
 	type AssetRegistry = OrmlAssetRegistry;
 	type Balance = Balance;
+	type BalanceRatio = Quantity;
```

Comment:
I named it as `BalanceRatio` because it is how the generic is called in some places in pool_system. But I do not feel well with any naming 😆, neither `Quantity` or `Ratio`. Probably we should refactor this with better naming in another PR.